hack/update-codegen.sh

```diff
@@ -138,3 +138,20 @@ CODEGEN_PKG=./vendor/k8s.io/code-generator vendor/k8s.io/sample-controller/hack/
 CODEGEN_PKG=./vendor/k8s.io/code-generator vendor/k8s.io/apiextensions-apiserver/hack/update-codegen.sh
 CODEGEN_PKG=./vendor/k8s.io/code-generator vendor/k8s.io/metrics/hack/update-codegen.sh
 CODEGEN_PKG=./vendor/k8s.io/code-generator vendor/k8s.io/apiextensions-apiserver/examples/client-go/hack/update-codegen.sh
+
+# call generation on admission controllers for now
+
+# resourcequota admission controller
+vendor/k8s.io/code-generator/generate-internal-groups.sh deepcopy,conversion,defaults \
```

Comment:
deepcopy and conversion (not sure about defaults) should be generated by Makefile.generated_files. Doesn't that work?

Reply:
cc @thockin